src/Data/Bit/Immutable.hs

```diff
@@ -189,10 +189,10 @@ cloneToWords v = runST $ do
   U.unsafeFreeze w
 {-# INLINE cloneToWords #-}
 
--- | Cast a unboxed vector of 'Word8'
+-- | Cast an unboxed vector of 'Word8'
 -- to an unboxed vector of bits.
 --
--- On big-endian architectures 'castFromWords8'
+-- On big-endian architectures 'castFromWords8', this
```

Comment:
I don't understand this change, it does not make sense to me.

Reply:
The "this" refers to the function itself (`castFromWords8`), but I can change it back if you find that easier to read.